pydantic/json_schema.py

```diff
@@ -148,7 +220,26 @@ def generate_definitions(
         self, inputs: Sequence[tuple[JsonSchemaKeyT, JsonSchemaMode, core_schema.CoreSchema]]
     ) -> tuple[dict[tuple[JsonSchemaKeyT, JsonSchemaMode], DefsRef], dict[DefsRef, JsonSchemaValue]]:
         """
-        Given a list of core_schema, generate all JSON schema definitions, and return the generated definitions.
+        Given a list of core_schema, generates all JSON schema definitions from a list of core schemas, and
+        returns the generated definitions.
+
+        Args:
+            inputs (Sequence[tuple[JsonSchemaKeyT, JsonSchemaMode, core_schema.CoreSchema]]): A sequence of tuples,
+                where:
+
+                - `JsonSchemaKeyT` is a JSON schema key type.
```

Comment:
This is kind of weird — the JsonSchemaKeyT is a generic type that can be anything the user wants; the idea is that the first returned item will have keys consisting in part of these items. The description of the `Returns:` below basically explains this, but I think it's also phrased a bit confusingly there.

I might describe it here more like  — `'JsonSchemaKeyT' will be paired with the JsonSchemaMode to form the keys of the first returned dictionary`